simulariumio/tests/converters/test_springsalad_converter.py

```diff
@@ -1,575 +1,419 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 
+import numpy as np
 import pytest
 
 from simulariumio.springsalad import SpringsaladConverter, SpringsaladData
 from simulariumio import DisplayData, MetaData, InputFileData, JsonWriter
 from simulariumio.constants import (
     DEFAULT_CAMERA_SETTINGS,
-    CURRENT_VERSION,
     DISPLAY_TYPE,
+    DEFAULT_BOX_SIZE,
+    VIZ_TYPE,
 )
 
 
+data = SpringsaladData(
+    sim_view_txt_file=InputFileData(
+        file_path=("simulariumio/tests/data/springsalad/test.txt"),
+    ),
+    draw_bonds=False,
+)
+converter = SpringsaladConverter(data)
+results = JsonWriter.format_trajectory_data(converter._data)
+
+
+# test box data default
+@pytest.mark.parametrize(
+    "box_size, expected_box_size",
+    [
+        (
+            results["trajectoryInfo"]["size"],
+            {
+                "x": DEFAULT_BOX_SIZE[0],
+                "y": DEFAULT_BOX_SIZE[1],
+                "z": DEFAULT_BOX_SIZE[2],
+            },
+        )
+    ],
+)
+def test_box_size_default(box_size, expected_box_size):
```

Comment:
I am pretty certain that all these defaults are set by the `TrajectoryConverter`? I think for these package specific converters, we should be testing that it gets data from specific format => trajectory data, I think that's `converter._data`. But we don't need to re-test all this other stuff that we already know works because it works in `TrajectoryConverter`

Reply:
Box size was weird for me because unlike some other parts of metadata, if nothing is provided, the specific converter has to call `meta_data._set_box_size()` for the default box size to be put in. Of course, for proper unit tests we should be testing `_set_box_size()` independently, but I do want to make sure that the default case doesn't result in breaking behavior in any of the specific converters. That being said, I could pull out the tests for other default tests (like camera position)